.github/actions/check_for_gftl/action.yml

```
@@ -9,11 +9,11 @@ runs:
     - name: check gFTL installation
       shell: python
       run: |
-        import pyccel
-        from os.path import exists, join
-        pyccel_loc = pyccel.__path__[0]
-        print(pyccel_loc)
-        assert exists(join(pyccel_loc,'extensions/gFTL/install/GFTL-1.13/include/v2'))
-        if ${{ inputs.not_editable }}:
-          assert not exists(join(pyccel_loc,'extensions/gFTL/doc'))
+        import importlib.resources
+        gftl_folder = importlib.resources.files(f'pyccel.extensions.gftl_install')
+        gftl_folder_gen = importlib.resources.as_file(gftl_folder)
+        with gftl_folder_gen as f:
```

Since `importlib.resources.as_file` returns a context manager, I think it is safer to call it in the `with` block:

```suggestion
        with importlib.resources.as_file(gftl_folder) as f:
```